docs/concepts/configuration/pod-priority-preemption.md

```diff
@@ -0,0 +1,177 @@
+---
+approvers:
+- davidopp
+- wojtek-t
+title: Pod Priority and Preemption
+---
+
+[Pods](/docs/user-guide/pods) in Kubernetes 1.8 and later can have priority. Priority
+indicates the importance of a pod. When a pod cannot be scheduled, scheduler tries
+to preempt lower priority pods in order to make scheduling of the pending pod possible.
+
+* TOC
+{:toc}
+
+## How to use it
+In order to use priority and preemption in Kubernetes 1.8, you should follow these
+steps:
+
+1. Enable Priority and Preemption.
+1. Add one or more PriorityClasses.
+1. Create pods with `PriorityClassName` set to one of the added PriorityClasses.
+
+The following sections provide more information about these steps.
+
+## Enable Priority and Preemption
```

Comment:
Can you please also mention that we should enable admission controller plugin here?

Reply:
correct, there has to be `--admisstion-control=...,Priority` command line option.

Reply:
Yes, but we should always have `ResourceQuota` as last one.

Reply:
Also need `--runtime-config=scheduling.k8s.io/v1alpha1=true`, right? This could have worked because we are enabling this alpha API by default in current code base. However, the convention was to have all alpha APIs disabled by default, IIRC.

Reply:
Hmm, I think that we will not hit https://github.com/kubernetes/kubernetes/blob/master/plugin/pkg/admission/priority/admission.go#L148 the featuregate of priority if does not enable the admission control? @bsalamat

Reply:
@bsalamat can you please show me the detail where did we add it to the list of default admission controllers? I did not find the code where it was added, am I missing anything?

Reply:
In this PR: https://github.com/kubernetes/kubernetes/pull/49322/files
I am not sure if it was the right thing to do given the downgrade issue we are observing.

Reply:
Ah, I see, thanks @bsalamat , I think that when downgrade, the downgrade script should have some logic to delete this admission control plugin.

Reply:
That's what I would expect as well, but I am not sure if the downgrade process actually removes them, otherwise #52226 shouldn't have happened.